www/artists/dloran/work.html

```diff
@@ -61,9 +61,6 @@
                         This is a compilation video of David doing some of his many works of art
```

Comment:
Should we really be saying "This is a compilation video" when we're deleting the video? That seems to be lying.